cliquet/resource.py

```diff
@@ -278,6 +278,9 @@ def __init__(self, request, context):
         logger.bind(collection_id=self.collection.collection_id,
                     collection_timestamp=self.timestamp)
 
+    def get_parent_id(self, request):
```

Comment:
Because you need the request in order to get the request and get the parent_id :)